com.unity.render-pipelines.high-definition/Documentation~/Decal-Shader.md

```diff
@@ -4,33 +4,44 @@
 
 ## Properties
 
+### Surface Options
+
+These properties allow you to set the affected attributes of the Material the decal is project onto when HDRP renders it into the Scene.
+
+| **Property**                    | **Description**                                              |
+| ------------------------------- | ------------------------------------------------------------ |
+| **Affect BaseColor**            | Enable the checkbox to make this decal use the **baseColor** properties. Otherwise the decal has no baseColor effect. Either disabled or enabled, the alpha channel of the base color is still use as an opacity for its other properties. |
```

Comment:
Regardless of whether you enable or disable this property, HDRP still uses the alpha channel of the base color as an opacity for the other properties.